src/test/kotlin/org/rust/ide/inspections/RsMainFunctionNotFoundInspectionTest.kt

```diff
@@ -0,0 +1,85 @@
+/*
+ * Use of this source code is governed by the MIT license that can be
+ * found in the LICENSE file.
+ */
+
+package org.rust.ide.inspections
+
+class RsMainFunctionNotFoundInspectionTest : RsInspectionsTestBase(RsMainFunctionNotFoundInspection::class) {
+
+    fun `test does not exist main function`() = checkErrors("""
```

Comment:
Maybe it's better to use `checkByFileTree` for all tests to be consistent and explicitly show file name because it's important here?

Reply:
Yes. It may be better to explicitly indicate the file name.